src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json

```diff
@@ -60,6 +60,12 @@
             "Areas/**"
           ]
         },
+        {
+          "condition": "(IndividualB2CAuth)",
+          "rename": {
+            "Pages/_LoginPartial.cshtml" : "Pages/Shared/_LoginPartial.cshtml"            
```

Comment:
I would have them living side by side in the `Pages/Shared` folder, name each of them explicitly `_LoginPartial.Identity.cshtml` and `_LoginPartial.OrgAuth.cshtml` and just make the rename be from `_LoginPartial.Identity.cshtml -> _LoginPartial.cshtml` and from `_LoginPartial.OrgAuth.cshtml -> _LoginPartial.cshtml`

The reason for it is that is not obvious what case each _LoginPartial file is for when you are editing them

Reply:
That sounds like a good idea to me.